syncnotification.go

```diff
@@ -50,6 +50,104 @@ func (mw *MultiWallet) handlePeerCountUpdate(peerCount int32) {
 	}
 }
 
+// Fetch CFilters Callbacks
+
```

Comment:
Comment can be more descriptive

Reply:
That comment is used to mark the beginning of cfilters callbacks. Similar comments are used to mark other sync stages;
https://github.com/planetdecred/dcrlibwallet/blob/38838d9adce251fe25d679fdc73c77ec87ac5898/syncnotification.go#L151 https://github.com/planetdecred/dcrlibwallet/blob/38838d9adce251fe25d679fdc73c77ec87ac5898/syncnotification.go#L310
https://github.com/planetdecred/dcrlibwallet/blob/38838d9adce251fe25d679fdc73c77ec87ac5898/syncnotification.go#L455